reminder/module.py

```diff
@@ -0,0 +1,465 @@
+            value=_(ctx, "✅ for confirmation") + _(ctx, "❎ for storno"),
+            inline=False,
+        )
+
+        message = await ctx.send(embed=embed)
+
+        await message.add_reaction("✅")
+        await message.add_reaction("❎")
+
+        def check_id(reaction, user_id):
+            return (
+                reaction.message.id == message.id
+                and (str(reaction.emoji) == "✅" or str(reaction.emoji) == "❎")
+                and user_id.id == query.remind_id
+            )
+
+        try:
+            reaction, user = await self.bot.wait_for(
+                "reaction_add", check=check_id, timeout=60
+            )
+        except asyncio.TimeoutError:
+            await ctx.send(_(ctx, "Deleting timed out."))
+        else:
+            if str(reaction.emoji) == "✅":
+                await bot_log.debug(
+                    ctx.author,
+                    ctx.channel,
+                    f"Deleting reminder from db - ID: {query.idx}, time: {query.remind_date}, status: {query.status}",
+                )
+                query.delete()
+            elif str(reaction.emoji) == "❎":
+                ctx.send(_(ctx, "Deletion aborted."))
+        try:
+            await message.delete()
+        except (nextcord.errors.NotFound, nextcord.errors.Forbidden):
+            pass
+
+    # HELPER FUNCTIONS
+
+    async def _process_text(
+        self, ctx: commands.Context, datetime_str: str, text: Optional[str]
+    ):
+        if text is not None:
+            lines = ctx.message.content
+
+            lines = lines.split(datetime_str)[1]
+
+            lines = re.split(" ", lines)
+            while "" in lines:
+                lines.remove("")
+            text = " ".join(lines)
+
+            if len(text) > 1024:
+                text = text[:1024]
+                text = text[:-3] + "```" if lines.count("```") % 2 != 0 else lines
+
+        return text
+
+    async def _get_embed(self, tc, query):
+        reminder_user = await self._get_member(query.guild_id, query.remind_id)
+
+        if reminder_user is None:
+            reminder_user_name = "_(Unknown user)_"
+        else:
+            reminder_user_name = nextcord.utils.escape_markdown(
+                reminder_user.display_name
+            )
+
+        embed = utils.discord.create_embed(
+            author=reminder_user,
+            title=_(utx, "Reminder"),
+        )
+
+        if query.author_id != query.remind_id:
+            embed.add_field(
+                name=_(utx, "Reminded by"),
+                value=reminder_user_name,
+                inline=True,
+            )
+        if query.message != "":
+            embed.add_field(
+                name=_(utx, "Message"),
+                value=query.message,
+                inline=False,
+            )
+        embed.add_field(name=_(utx, "URL"), value=query.permalink, inline=True)
+
+        return embed
+
+    async def _send_reminder_list(self, ctx, query):
```

Comment:
does this need to be sent here? I would just return a list of table strings and deal with sending in the command code.

Partially because if in the future we make a scrollable message then it would be better done there

Reply:
Well, it's better for now since we have 2 commands doing exactly the same but with different query. It can be reworked in future as those commands would change.